eq-author/src/components/ContentPickerv3/MetadataPicker.test.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { render, fireEvent } from "tests/utils/rtl";
+import { ANSWER, METADATA } from "../ContentPickerSelectv3/content-types";
+import Theme from "contexts/themeContext";
+
+import MetadataPicker from "./MetadataPicker";
+
+describe("Metadata Picker", () => {
+  let props;
+  const setContentType = jest.fn();
+
+  beforeEach(() => {
+    props = {
+      data: [
+        {
+          id: "91d8a289-e625-4b94-869b-a7d698be2294",
+          key: "ru_name",
+          alias: "Ru Name",
+          type: "Text",
+          textValue: "ESSENTIAL ENTERPRISE LTD.",
+        },
+        {
+          id: "74c9ca55-dc3b-46da-b5eb-5eca48c4f443",
+          key: "ref_p_start_date",
+          alias: "Start Date",
+          type: "Date",
+          dateValue: "2016-05-01",
+        },
+      ],
+      contentType: ANSWER,
+      contentTypes: [ANSWER, METADATA],
+      setContentType,
+      isSelected: jest.fn(),
+      onSelected: jest.fn(),
+      isSectionSelected: jest.fn(),
+    };
+  });
+  const renderMetadataPicker = () =>
+    render(
+      <Theme>
+        <MetadataPicker {...props} />
+      </Theme>
+    );
```

Comment:
```suggestion
  beforeEach(() => {
    props = {
      data: [
        {
          id: "91d8a289-e625-4b94-869b-a7d698be2294",
          key: "ru_name",
          alias: "Ru Name",
          type: "Text",
          textValue: "ESSENTIAL ENTERPRISE LTD.",
        },
        {
          id: "74c9ca55-dc3b-46da-b5eb-5eca48c4f443",
          key: "ref_p_start_date",
          alias: "Start Date",
          type: "Date",
          dateValue: "2016-05-01",
        },
      ],
      contentType: ANSWER,
      contentTypes: [ANSWER, METADATA],
      setContentType,
      isSelected: jest.fn(),
      onSelected: jest.fn(),
      isSectionSelected: jest.fn(),
    };
  });
  
  
  afterEach(() => {
    jest.clearAllMocks();
  });
  
  const renderMetadataPicker = () =>
    render(
      <Theme>
        <MetadataPicker {...props} />
      </Theme>
    );
```

The mock function `setContentType` is not currently being cleared after each of the tests. This means in the last test (should display metadata list), the function call test is passing as the test run before it also called the `setContentType` function with `METADATA` - moving the expect statement on line 61 to line 58 would still pass from the previous test.

Adding the suggested code will clear out the mock functions after each test is run, meaning the mock function will have to be called within each test for the test to pass.